src/NZGBplugin/LINZ/gazetteer/Database.py

```diff
@@ -24,14 +24,15 @@
 from sqlalchemy.pool import Pool
 from sqlalchemy.sql import text
 
-_host = os.environ.get("GAZETTEER_DBHOST") or "prdgeo01"
-_port = os.environ.get("GAZETTEER_DBPORT") or "5432"
-_database = os.environ.get("GAZETTEER_DB") or "gazetteer"
-_schema = os.environ.get("GAZETTEER_DBSCHEMA") or "gazetteer"
-_user = os.environ.get("GAZETTEER_DBUSER") or getpass.getuser()
-_password = os.environ.get("GAZETTEER_DBUSERPASS") or None
+_host = os.environ.get("PGHOST") or "prdgeo01"

```

Comment:
You are changing behaviour of the plugin here, it should never be needed, when adding testcases, or should be well justified...

Reply:
I altered these to match the docker envi var names to that the plugin will use these vars. 

In hindsight I should have changed the docker envi var names but a grep shows nothing else depends on them.

Reply:
I'm not sure (not being a plugin user) but I'd guess _USERS_ could depend on that. Say you are a user, and you have GAZETTEER_XXXX variables in your .profile file. When upgrading from old plugin to new plugin you loose honouring of those env variable. Can you please verify if that's the case or not ?